lib/airbrake-ruby/config.rb

```diff
@@ -167,6 +176,15 @@ def logger=(logger)
       @logger = logger || @logger
     end
 
+    # Sets the default method wrapping style. Must be :chain or :prepend.
+    def default_wrapping_style=(style)
+      style = style.to_sym if style.respond_to?('to_sym')
+      unless %i[chain prepend].include?(style)
+        raise ArgumentError "default_wrapping_style must be :chain or :prepend"
```

Comment:
```suggestion
        raise ArgumentError, "default_wrapping_style must be :chain or :prepend"
```

Reply:
That's what I get for not writing a test for that line.